live-site/src/pages/index.tsx

```diff
@@ -21,6 +22,8 @@ const IndexPage: React.FC = () => {
   return (
     <div>
       <Header tabs={liveSiteTabInfo} isDay={true} />
+      <h1>hello</h1>
+      {Date.now() && <HackingRemaining />}
```

Comment:
add a desktop check to meet ticket requirement (tho personally I disagree with it?)

Reply:
^^ I think I did confirm with sarah and she said it shouldn't be on the mobile version so under tablet size stop rendering this

Reply:
@Dhivas-Sugumar could you please confirm with sarah if the time remaining should be on mobile? I think she said it shouldn't but I can't remember too clearly